Doc/library/csv.rst

```diff
@@ -275,6 +275,8 @@ The :mod:`csv` module defines the following classes:
       reflecting the parameters found.  If the optional *delimiters* parameter
       is given, it is interpreted as a string containing possible valid
       delimiter characters.
+      If not excluded by the *delimiters* parameter, and if there is ambiguity,
```

Comment:
Why is the documentation here not added to the `def sniff` docstring? That would be a natural place for documentation that is available from the command line or IDE.

Reply:
I'm not sure what information should go into the docstring vs in the docs. I imagine there's some duplicate, but surely not everything in the docs goes into the docstring and vice-versa. The docstring of `sniff` is looking a bit empty though, so it could use some work for sure.